app/views/hubstats/users/show.html.erb

```diff
@@ -4,6 +4,16 @@
     <!-- Title with the user's GitHub username -->
     <h1 class="title text-center"><a href=<%=@user.html_url%>> <%= @user.login %></a> </h1>
 
+    <!-- Show what team(s) the user is on -->
+    <h4 class="text-center">
+      <% if @user.teams.where(hubstats: true).first == nil %> <!-- first because user is only part of one valid team -->
```

Comment:
You have logic leaking into your view.  You should use the existing method on the User model instead, `@user.team`, in your conditional here and in the output below.